components/sidebar/sidebar.jsx

```diff
@@ -478,16 +490,82 @@ export default class Sidebar extends React.PureComponent {
         );
     }
 
+    renderOrderedChannels = () => {
+        const {orderedChannelIds} = this.props;
+
+        const channelsToHide = ['unreads', 'favorite'];
+
+        const orderedChannelSections = orderedChannelIds.map((section) => {
```

Comment:
Maybe we could combine these two map statements since we're going over the same list twice?